test/cluster-isolation/mzcompose.py

```diff
@@ -179,7 +179,7 @@ def validate(c: Composition) -> None:
   FORMAT BYTES
 
 > SELECT * FROM source1
-A 1
```

Comment:
I have a feeling the offsets are providing a bit of test coverage for the offsets feature. Might be better to instead explicitly include the offset in the source definition? For this and all the other tests.

Reply:
There's a good set of tests of `INCLUDE OFFSET` and `INCLUDE OFFSET AS ...` in kafka-avro-sources.td that verifies the offset behavior with multipartition Kafka topics too, so I felt comfortable excising it from pretty much everywhere else! (I left it in a few sink tests where it seemed worthwhile to verify the offsets in exactly matched the offsets out.)

Reply:
makes sense! 🙌